source/jormungandr/jormungandr/interfaces/parsers.py

```diff
@@ -36,6 +36,13 @@ def depth_argument(value, name):
     return int(value)
 
 
+def speed_value(value, name):
+    if float(value) <= 0:
```

Comment:
avoid double conversion:
value = float(value)

better:
message = "The {} argument must be float > 0, you gave : {}"
try:
    value = float(value)
except ValueError:
    raise ValueError(message.format(name, value))
if value <= 0:
    raise ValueError(message.format(name, value))

et ajouter 'x' dans la liste du test unitaire